man/ru/apol.1

```diff
@@ -0,0 +1,43 @@
+.\" Copyright (c) 2016 Tresys Technology, LLC.  All rights reserved.
+.TH apol 1 2016-02-20 "Tresys Technology, LLC" "SETools: утилиты анализа политики SELinux"
```

Comment:
Here and below, it should be `"SELinux Project"` instead of `"Tresys Technology, LLC"`.